compiler/rustc_codegen_ssa/src/back/archive.rs

```diff
@@ -233,6 +233,7 @@ impl<'a> ArArchiveBuilder<'a> {
             "bsd" => ArchiveKind::Bsd,
             "darwin" => ArchiveKind::Darwin,
             "coff" => ArchiveKind::Coff,
+            "aix_big" => ArchiveKind::AixBig,
```

Comment:
You will also need to add it to LlvmArchiveWriter. Due to some regressions it was necessary to revert back to it.

https://github.com/rust-lang/rust/blob/ad8e1dc2863f63c35ef3ceef3064d0851a1d2582/compiler/rustc_codegen_llvm/src/llvm/mod.rs#L131-L143

Reply:
Seems definitions in FFI doesn't match those from LLVM?

https://github.com/rust-lang/rust/blob/0d32c8f2ce10710b6560dcb75f32f79c378410d0/compiler/rustc_codegen_llvm/src/llvm/ffi.rs#L545-L553

Reply:
https://github.com/rust-lang/llvm-project/blob/477e7285b12f876ad105188cfcfc8adda7dc29aa/llvm/include/llvm/Object/Archive.h#L339